src/components/common/ConfirmModal.tsx

```diff
@@ -45,8 +40,8 @@ const dialogStyles = makeStyles({
   },
   dialogButton: {
     color: Colours.Secondary,
-    height: 48,
-    minWidth: 107,
+    height: '48px',
+    minWidth: '107px',
```

Comment:
Small comment: Is it weird that this here is using the `minWidth` property while the `dialogDelete` and `dialogConfirm` styles use `width`?

Reply:
we can leave it in for now - when we get around to cleaning up modal styling, we can address it then

([added to this ticket](https://www.notion.so/uwblueprintexecs/dialog-modal-cleanup-4e0ec85b6e2f41ef8829c53a8a0fa134))